db/seeds.rb

```diff
@@ -1,21 +1,22 @@
 jcasimir = User.create(username: "jcasimir", password: "test", email: "jcasimir@example.com", name: "Jeff Casimir", address: "1510 Blake St", city: "Denver", state: "CO", zip: "80111")
 admin = User.create(username: "admin", password: "admin", role: 1, email: "jcasimir@example.com", name: "Jeff Casimir", address: "1510 Blake St", city: "Denver", state: "CO", zip: "80111")
 
+date = DateTime.now
 storage = Category.create(name: "storage")
-item_1 = storage.items.create(name: "Laser Disc", description: "Make great plates.", price: "5.99", image: "https://visualhunt.com/photos/m/5/laser-and-video-discs-24.jpg", end_time: DateTime.at_beginning_of_day, status: 0)
-item_2 = storage.items.create(name: "Floppy Disc", description: "Need to store a whopping 1.44 MB of data? This bad boy has got you covered.", price: "5.99", image: "https://visualhunt.com/photos/m/5/day-037-did-floppy-exist-once.jpg", end_time: DateTime.at_beginning_of_day, status: 0)
-item_3 = storage.items.create(name: "CD", description: "Like ninja stars but better.", price: "2.99", image: "https://visualhunt.com/photos/m/4/cd-dvd-computer-data-shiny-digital-disk.jpg", end_time: DateTime.at_beginning_of_day, status: 0)
+item_1 = storage.items.create(name: "Laser Disc", description: "Make great plates.", price: "5.99", image: "https://visualhunt.com/photos/m/5/laser-and-video-discs-24.jpg", end_time: date.prev_day, status: 0)
```

Comment:
We're planning on having auctions close on the hour using heroku's scheduler, but this will produce end-times on whatever minute and second combo the code is run. It should pass our checks now, because we're only checking for a time that's current or past. But we need to be aware of it if/when we put validations in later on.